frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -598,12 +598,14 @@ final class BloopBspServices(
                 canTest = true,
                 canRun = true
               )
+              val isJavaOnly = p.scalaInstance == ScalaInstance.scalaInstanceFromBloop(bspLogger)(ioScheduler)
```

Comment:
What do you think about renaming `scalaInstanceFromBloop` to `scalaInstanceForJavaProjects` so that it's clearer for future readers that we assign this instance to all java projects?